Improve layout of settings & commands tables

Robyt3 opened the pull request:
- Increase table cell padding.
- Add consistent table border.
- Scale table to 100% width.
- Remove inconsistent margins on left and right side.
- Add background color for hovered table rows.
- Add style for `colorpreview` class which will be used to show a preview for default color values.

Screenshots:
- Before: 
![image](https://github.com/user-attachments/assets/9cb2e80b-0626-48f2-8c2d-5571d0b438d2)

- After: 
![image](https://github.com/user-attachments/assets/8764be3d-19ee-44f9-bb26-dcc6bd3cfc02)

def- reviewed (commented):
Everything looks nice to me except the table border. If there are too many borders it looks too busy.
(I think I originally got this inspiration from Tufte's The Visual Display of Quantitative Information)

Robyt3 commented:
> Everything looks nice to me except the table border. If there are too many borders it looks too busy. (I think I originally got this inspiration from Tufte's The Visual Display of Quantitative Information)

I decreased the border width to 1px again and removed the inside vertical borders.

I also added the `colorpreview` class to soon use in an updated script for generating the tables, to preview the default color values like so:

![image](https://github.com/user-attachments/assets/118a6258-56e8-41ac-9d96-839ecfb54a37)
